README.md

```diff
@@ -25,6 +25,7 @@ We follow a [Code of Conduct](docs/code-of-conduct.md).
     * [How to use different PHP versions](docs/how-to-use-different-php-versions.md)
 		* [Hosts and file structure](docs/hosts-and-file-structure.md)
 		* [Sharing with the world](docs/sharing-with-the-world-via-ngrok.md)
+		* [Implement a new or old PHP version](docs/installing-unsupported-php-versions.md)
```

Comment:
I suggest renaming this entry as the file and entry name are not the same. 
Something along the line of "Install other PHP versions" for both.